app/services/open_library_service.rb

```diff
@@ -41,6 +35,10 @@ def build_response(data)
     }
   end
 
+  def build_bad_response
+    { status: 404, response: { message: 'Book not found' } }
```

Comment:
When having an error, there shouldn't be a "response" but instead an "errors" as the frontend will check for presence of "error" or "result/response" sometimes, not just code.

https://github.com/Wolox/tech-guides/blob/master/backend/backend-standard-and-best-practices.md